pymetacg/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+include(nanobind)
+include(spdlog)
+include(json)
+
+nanobind_build_library(nanobind-static)
+
+# nanobind_add_module(pymetacg NOSTRIP pymetacg.cpp) We use nanobind's low-level API, as the high-level API sets the
```

Comment:
Leftover from experimenting? 
Comment is very good, just formatting seems a bit off